lib/kubernetes-deploy/deploy_task.rb

```diff
@@ -319,6 +324,12 @@ def record_invalid_template(err:, filename:, content: nil)
       @logger.summary.add_paragraph(debug_msg)
     end
 
+    def record_warnings(warning:, filename:)
```

Comment:
Do we need to support this for any other task? For example, we have access to warnings here as well:

https://github.com/Shopify/kubernetes-deploy/blob/3d5cf5964932658a91e3e5621719ffaaa7ee0aac/lib/kubernetes-deploy/runner_task.rb#L40